### PR TITLE
fix(home/battle) css/spacing fixes + re-factor

### PR DESCRIPTION
**Pull request by j-masci**
Home page had a lot of space on right at certain resolutions due to many different components having their own padding on the right. Seemed hard to fix as is, so I made a new component for a page with sidebar on right. I also re-factored the battle page to use this component.

### CODE REVIEW

**Comment by j-masci** [comment]
gonna look into doing this with mui grid so hold off on merge

File: src/pages/home/index.js [hunk 1]
```diff
@@ -14,48 +17,22 @@ export default function Home() {
 
   return (
     <Layout t="Home">
-      <Grid container spacing={3}>
-        {!loggedIn && (
-          <Grid container item xs={12} sm={12} spacing={3}>
```
Comment (metsavir) [inline src/pages/home/index.js:19]: ->`<Grid container item xs={12}>`

File: src/pages/home/index.js [hunk 2]
```diff
@@ -14,48 +17,22 @@ export default function Home() {
...
-            <Grid item xs={12}>
-              <WelcomeCard />
-            </Grid>
-          </Grid>
-        )}
-        <Grid
-          container
-          item
-          xs={12}
-          sm={7}
```
Comment (metsavir) [inline src/pages/home/index.js:29]:
```
xs={12}
sm={7}
```
-> `sm={7}`

File: src/pages/home/index.js [hunk 3]
```diff
@@ -14,48 +17,22 @@ export default function Home() {
...
-          <Grid item xs={12}>
-            <ReplaysCard />
-          </Grid>
-        </Grid>
-        <Grid
-          container
-          spacing={3}
-          item
-          xs={12}
-          sm={5}
```
Comment (metsavir) [inline src/pages/home/index.js:45]:
```
xs={12}
sm={5}
```
-> `sm`

**Review by metsavir — COMMENTED** [review]
Found a way to fix it without custom component, check comments.

**Pull request closed without merging**